### ISSUE TITLE
refactor: Self utilizing ping rework 

### ISSUE THREAD

**Issue by ethan-davies** [ethan-davies/mock#3]
For the most part the `ping` command does work however, if you are trying to ping something you would use:
`ping 8.8.8.8`, in this case we will be using Google. The expected output should be at least a couple of milliseconds but instead it ends up always being `0s`:
```
Ping statistics for 8.8.8.8:
  Packets: Sent = 0, Received = 0, Lost = 0 (NaN% loss)
Approximate round trip times:
  Minimum = 0s, Maximum = 0s, Average = 0s
```

**Comment by ethan-davies** [comment]
Issue has been temporarily fixed but as stated in the PR it will _NOT_ be permanent. 
* [Temporarily fixed ping command #7](https://github.com/ethan-davies/mock/pull/7)

**Comment by ethan-davies** [comment]
Rework has been confirmed on this issue. As seen in the [project roadmap](https://github.com/users/ethan-davies/projects/2)!
